#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "object_follower"

gen = ParameterGenerator()

gen.add("goal_dist_from_obj", double_t, 0,
        "Distance from object to goal", 1)

gen.add("object_frame",    str_t,    0, "Object frame",  "object")
gen.add("enable_following",   bool_t,   0, "Enable Following",  True)

exit(gen.generate(PACKAGE, "dynamic_tutorials", "Tutorials"))
